botprise/metadata.csv

```diff
@@ -0,0 +1,4 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+system.cpu.user,gauge,,,,The percent of time the CPU spent running user space processes.,,system,
+system.cpu.system,,,,,,,,
+system.cpu.i,,,,,,,,
```

Comment:
metrics should have a `botoprise.` prefix, type of metric, integration, description and short name are required fields

Reply:
@hithwen Thanks for reviewing the changes. Regarding metrics we have been using predefined metric to collect metrics from added server/client. Do we still need to populate values for metrics?